staging/src/k8s.io/library-go/SECURITY_CONTACTS

```diff
@@ -0,0 +1,17 @@
+# Defined below are the security contacts for this repo.
+#
+# They are the contact point for the Product Security Committee to reach out
+# to for triaging and handling of incoming issues.
+#
+# The below names agree to abide by the
+# [Embargo Policy](https://git.k8s.io/security/private-distributors-list.md#embargo-policy)
+# and will be removed and replaced if they violate that agreement.
+#
+# DO NOT REPORT SECURITY VULNERABILITIES DIRECTLY TO THESE NAMES, FOLLOW THE
+# INSTRUCTIONS AT https://kubernetes.io/security/
+
+cjcullen
```

Comment:
this is a stock list.  I'm willing to be in the list and @liggitt is a glutton for punishment.

Reply:
This should be kept in sync with [committee-product-security](https://github.com/nikhita/community/blob/284c616e100a19c184b79267dd838f69815b02c4/OWNERS_ALIASES#L152-L158).

Reply:
Synced